data/json/npcs/isolated_road/isolated_road_missions.json

```
@@ -0,0 +1,92 @@
+[
+  {
+    "id": "MISSION_BLACKSMITH_1",
+    "type": "mission_definition",
+    "name": { "str": "Find the gun" },
+    "description": "Cody wants you to go to a gun show and find a rare gun that Jay wanted to see.  It was an AK of some kind.",
```

The KORD is definitely AK-derived, but players might not be aware of that. Perhaps changing the phrase here would help clarify things.
```suggestion
    "description": "Cody wants you to go to a gun show and find a rare gun that Jay wanted to see.  It was an AK of some kind with a weird name.. something like CARD or CORE.",
```